src/test/java/org/jenkinsci/plugins/runselector/steps/SelectRunStepTest.java

```diff
@@ -115,7 +118,73 @@ public void specificBuildNumberDoesNotExist() throws Exception {
     }
 
-    private WorkflowRun createWorkflowJobAndRun(String script) throws Exception {
+    @Test
+    public void testStatusSymbol() throws Exception {
+        assumeSymbolDependencies();
+
+        WorkflowRun upstreamRun = createWorkflowJobAndRun("echo 'foobar'");
+        String projectName = upstreamRun.getParent().getFullName();
+        j.assertBuildStatusSuccess(upstreamRun);
+
+        WorkflowRun run = createWorkflowJobAndRun(format("" +
+                "def runWrapper = selectRun job: '%s', " +
+                " selector: status('Stable'), " +
+                " verbose: true", projectName));
+
+        j.assertBuildStatusSuccess(run);
+    }
+
+    @Test
+    public void testSpecificRunSymbol() throws Exception {
+        assumeSymbolDependencies();
+
+        WorkflowRun upstreamRun = createWorkflowJobAndRun("echo 'foobar'");
+        String projectName = upstreamRun.getParent().getFullName();
+        j.assertBuildStatusSuccess(upstreamRun);
+
+        WorkflowRun run = createWorkflowJobAndRun(format("" +
+                "def runWrapper = selectRun job: '%s', " +
+                " selector: specificRun('1'), " +
+                " verbose: true", projectName));
+
+        j.assertBuildStatusSuccess(run);
+    }
+
+    @Test
+    public void testPermalinkSymbol() throws Exception {
+        assumeSymbolDependencies();
+
+        WorkflowRun upstreamRun = createWorkflowJobAndRun("echo 'foobar'");
+        String projectName = upstreamRun.getParent().getFullName();
+        j.assertBuildStatusSuccess(upstreamRun);
+
+        WorkflowRun run = createWorkflowJobAndRun(format("" +
+                "def runWrapper = selectRun job: '%s', " +
+                " selector: permalink('lastStableBuild'), " +
+                " verbose: true", projectName));
+
+        j.assertBuildStatusSuccess(run);
+    }
+
+    /**
+     * To use the @Symbol annotation in tests, minimum workflow-cps version 2.10 is required.
+     * This dependency comes with other dependency version requirements, as stated by this method.
+     * To run tests restricted by this method, type
+     * <pre>
+     *  mvn clean install -Djenkins.version=1.642.1 -Djava.level=7 -Dworkflow-step-api.version=2.3 -Dworkflow-support.version=2.2 -Dworkflow-job.version=2.4 -Dworkflow-basic-steps.version=2.1 -Dworkflow-cps.version=2.10
+     * </pre>
+     */
+    private static void assumeSymbolDependencies() {
+        assumeThat(System.getProperty("jenkins.version"), startsWith("1.642"));
+        assumeThat(System.getProperty("java.level"), is("7"));
+        assumeThat(System.getProperty("workflow-step-api.version"), is("2.3"));
+        assumeThat(System.getProperty("workflow-support.version"), is("2.2"));
+        assumeThat(System.getProperty("workflow-job.version"), is("2.4"));
+        assumeThat(System.getProperty("workflow-basic-steps.version"), is("2.1"));
+        assumeThat(System.getProperty("workflow-cps.version"), is("2.10"));
```

Comment:
`hudson.util.VersionNumber` is useful for comparing versions (should be included in dependencies via jenkins-core).
https://github.com/jenkinsci/lib-version-number/blob/master/src/main/java/hudson/util/VersionNumber.java